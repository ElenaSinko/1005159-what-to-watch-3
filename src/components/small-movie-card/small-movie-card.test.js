import React from "react";
import renderer from "react-test-renderer";
import SmallMovieCard from "./small-movie-card.jsx";

const smallMovieCard = {
  title: `test`,
  img: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
};

it(`Render small movie card`, () => {
  const tree = renderer
    .create(<SmallMovieCard
      smallMovieCard={smallMovieCard}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
