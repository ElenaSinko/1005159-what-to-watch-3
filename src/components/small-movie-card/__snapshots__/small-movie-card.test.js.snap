// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render small movie card 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
>
  <a
    href="/film/undefined"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        className="player__video"
        height="175"
        muted={true}
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <span
        className="small-movie-card__link"
        href="movie-page.html"
        style={
          Object {
            "color": "white",
          }
        }
      />
    </h3>
  </a>
</article>
`;
